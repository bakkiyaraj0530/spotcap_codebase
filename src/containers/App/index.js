import React from 'react'
import { Route } from 'react-router-dom'
import CategoryView from '../CategoryView'
import Tagedetail from '../Tagedetail'
import AddRule from '../AddRule'

const App = () => (
  <div>
    <main>
      <Route exact path="/" component={CategoryView} />
      <Route exact path="/tagdetail-view" component={Tagedetail} />
      <Route exact path="/new-rule" component={AddRule} />
    </main>
  </div>
)

export default App
