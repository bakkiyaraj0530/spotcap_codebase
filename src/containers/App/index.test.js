import React from 'react';
import App from './index';
import { shallow, mount, render } from 'enzyme';

it('renders without crashing', () => {
  shallow(<App />);
});
