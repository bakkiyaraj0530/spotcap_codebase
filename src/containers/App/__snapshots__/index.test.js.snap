// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> Link matches snapshot 1`] = `
<a
  href="/someLocation"
  onClick={[Function]}
/>
`;

exports[`<App /> should render 1`] = `
<div
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  <header>
    <a
      href="/"
      onClick={[Function]}
    >
      Category View
    </a>
  </header>
  <main />
</div>
`;
